frontend/src/libraries/explorer-wamp/details.ts

```diff
@@ -0,0 +1,67 @@
+import { ExplorerApi } from ".";
+
+export interface Details {
+  accountsCount: number;
+  totalNodesCount: number;
+  onlineNodesCount: number;
+  lastDayTxCount: number;
+  lastBlockHeight: number;
+  transactionsPerSecond: number | null;
+}
+
+export default class DetailsApi extends ExplorerApi {
+  async getDetails(): Promise<Details> {
+    try {
+      const detail = await this.call("select", [
+        `SELECT
+          total_accounts.accountsCount,
+          nodes.totalNodesCount,
+          online_nodes.onlineNodesCount,
+          transactions.lastDayTxCount,
+          last_block.lastBlockHeight
+        FROM
+          (SELECT COUNT(*) as totalNodesCount FROM nodes) as nodes,
+          (SELECT COUNT(*) as onlineNodesCount FROM nodes WHERE last_seen > (strftime('%s','now') - 60) * 1000) as online_nodes,
+          (SELECT COUNT(*) as lastDayTxCount FROM transactions
+              WHERE block_timestamp > (strftime('%s','now') - 60 * 60 * 24) * 1000
+          ) as transactions,
+          (SELECT height as lastBlockHeight FROM blocks ORDER BY height DESC LIMIT 1) as last_block,
+          (SELECT COUNT(*) as accountsCount FROM accounts) as total_accounts`
+      ]).then((it: any) => it[0]);
+      const Tps = await this.getTPS();
+      return {
+        ...detail,
+        transactionsPerSecond: Tps
+      };
+    } catch (error) {
+      console.error("Details.getDetails failed to fetch data due to:");
+      console.error(error);
+      throw error;
+    }
+  }
+
+  async getTPS() {
```

Comment:
I believe, these queries can be merged into the main query just fine.

If you still decide to keep TPS separately (though, i cannot think of a reason), you should at least merge these two queries, so they don’t get out of sync (waiting on a slow network can create a noticeable delay between the two calls and that will become transaction per 10+ seconds)

Reply:
np! I separate it first try to see whether SQL query is separated correctly.

Reply:
Remove the dead function.